### PR TITLE
Support package.json source array

### LINKED ISSUE #364: TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type object
I'm getting this error with microbundle v0.7.0 - v0.11.0 and npm, but not with v0.6.0 or older and npm (those give #74, so microbundle seems unusable with npm)..

```
git clone git@github.com:draftbit/twitter-lite.git
cd twitter-lite
git checkout 5a8c7f96cdbce1aa6057231b326433556b35afc2
# edit package.json and change the version of microbundle to 0.11.0
npm install
```

Output:

```
npm WARN deprecated kleur@2.0.2: Please upgrade to kleur@3 or migrate to 'ansi-colors' if you prefer the old syntax. Visit <https://github.com/lukeed/kleur/releases/tag/v3.0.0\> for migration path(s).
npm WARN deprecated browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.

> iltorb@2.4.2 install /home/dandv/prg/twitter-lite/node_modules/iltorb
> node ./scripts/install.js || node-gyp rebuild

info looking for cached prebuild @ /home/dandv/.npm/_prebuilds/f091be-iltorb-v2.4.2-node-v67-linux-x64.tar.gz
info found cached prebuild 
info unpacking @ /home/dandv/.npm/_prebuilds/f091be-iltorb-v2.4.2-node-v67-linux-x64.tar.gz
info unpack resolved to /home/dandv/prg/twitter-lite/node_modules/iltorb/build/bindings/iltorb.node
info unpack required /home/dandv/prg/twitter-lite/node_modules/iltorb/build/bindings/iltorb.node successfully
info install Successfully installed iltorb binary!

> husky@1.3.1 install /home/dandv/prg/twitter-lite/node_modules/husky
> node husky install

husky > setting up git hooks
husky > done

> twitter-lite@0.9.3 prepare /home/dandv/prg/twitter-lite
> microbundle

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type object

    at validateString (internal/validators.js:125:11)
    at Object.resolve (path.js:977:7)
    at getInput (/home/dandv/prg/twitter-lite/node_modules/microbundle/dist/microbundle.js:342:45)
    at /home/dandv/prg/twitter-lite/node_modules/microbundle/dist/microbundle.js:433:30

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! twitter-lite@0.9.3 prepare: `microbundle`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the twitter-lite@0.9.3 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/dandv/.npm/_logs/2019-03-26T05_49_03_443Z-debug.log
```

### PR DESCRIPTION
**Pull request by maraisr**
With this PR, I am to solve issue #364, however I believe this shouldn't also need to support `source` that could be either a directory, or a file, or a glob. To keep things simple.

fixes #364

### CODE REVIEW

**Comment by maraisr** [comment]
Come to think of it @developit I'm having second thoughts on this PR... Keen to hear your thoughts.

Even though it solves that issue #364, it does mimic the `first arg`, and therefor you do _kinda_ get an array's behaviour, with `"source": "./{a,b}.js"` also.

File: src/index.js [hunk 1]
```diff
@@ -306,7 +306,10 @@ async function getInput({ entries, cwd, source, module }) {
 		.concat(
 			entries && entries.length
 				? entries
-				: (source && resolve(cwd, source)) ||
+				: (source &&
+						(Array.isArray(source)
+							? source
+							: [source].map(file => resolve(cwd, file)))) ||
```
Comment (ForsakenHarmony) [inline src/index.js]: I think you meant to do `(Array.isArray(source) ? source : [source]).map()`?

Comment (maraisr) [reply src/index.js]: Ah. Yes.

**Comment by maraisr** [comment]
@ForsakenHarmony this one is ready again for another review! 👍

**Pull request merged**